packaging/userspace/userspace-packager.py

```diff
@@ -301,18 +309,33 @@ def _main():
 
 	pprint(vars(options))
 
+	if not lief_imported:
+		l.warning('lief cannot be imported. Will not be able to remove DT_NEEDED entries for ' +
+		          'unneeded libraries or set DF_ORIGIN flags. This can result in additional ' +
+		          'libraries being included in the package, and may also result in ' +
+		          'LD_LIBRARY_PATH needing to be set to run programs from the package.')
+		if options.chrpath_path is None:
+			options.chrpath_path = shutil.which('chrpath', path=env['PATH'])
+			if options.chrpath_path is None:
+				l.error('chrpath tool cannot be found. Without either lief or chrpath, will ' +
+				        'not be able to set rpath/runpath in package binaries. This will ' +
+				        'result in bad runpath/rpath values and LD_LIBRARY_PATH needing to be ' +
```

Comment:
you switched the order of rpath and runpath in these two lines.   pick an order, run with it.